pages/en/lb3/Polymorphic-relations.md

```diff
@@ -96,47 +138,140 @@ And:
-  foreignKey: 'imageableId', // defaults to 'as - Id'
-  discriminator: 'imageableType' // defaults to 'as - Type'
+Reader.hasMany(Picture, { polymorphic: {
+  foreignKey: 'imageableId',
+  discriminator: 'imageableType'
+  } 
+});
+
+// Alternative use `selector`
+
+Reader.hasMany(Picture, { polymorphic: {
+  selector: "imageable"
   } 
 });
 ```
 
-## HasManyThrough polymorphic relations
+## BelongsTo polymorphic relations
 
-To define a hasMany polymorphic relation, there must be a "through" model, similarly to a standard [HasManyThrough relation](HasManyThrough-relations.html).
+Because you define the related model dynamically, you cannot declare it up front.
+So instead of passing in the related model (name), you specify the name of the polymorphic relation.
 
-For example, "ImageLink":
+To define a belongsTo polymorphic relation, you need to provide the following parameters:
+
+- `type`: the relation type, in this case is 'belongsTo'
+- `as`: redefines **this** relation's name (optional)
+- ~~`model`~~: **NOT EXPECTED**: throws an error as relation validation
+- `polymorphic`:
+  - typeOf `polymorphic` === `Object`
+    - `foreignKey`:  A property of modelTo, representing the fk to modelFrom's id. 
+    - `discriminator`: A property of modelTo, representing the actual modelFrom to be looked up and defined dynamically.
+    - ~~`selector` or `as`~~: **NOT EXPECTED**: throws an error at relation validation
+  - typeOf `polymorphic` === `Boolean`
+      - `foreignKey` is generated as `relationName + 'Id'`,
+      - `discriminator` is generated as `relationName + 'Type'`
+
+{% include code-caption.html content="common/models/picture.json" %}
 
-{% include code-caption.html content="/common/models/ImageLink.json" %}
 ```javascript
 {
-  "name": "ImageLink",
+  "name": "Picture",
   "base": "PersistedModel",
   ...
   "relations": {
-    "author": {
+    "imageable": {
+      "type": "belongsTo",
+      "polymorphic": true
+    }
+  },
+...
+
+// Alternatively, use an object for setup
+
+{
+  "name": "Picture",
+  "base": "PersistedModel",
+  ...
+  "relations": {
+    "imageable": {
       "type": "belongsTo",
-      "model": "Author",
-      "foreignKey": "authorId"
-    },
-    "pictures": {
-      "type": "hasMany",
-      "model": "Picture",
       "polymorphic": {
         "foreignKey": "imageableId",
         "discriminator": "imageableType"
       }
     }
   },
 ...
-}
 ```
 
+Or, in code:
+
+{% include code-caption.html content="common/models/picture.js" %}
+```javascript
+Picture.belongsTo('imageable', {
+  polymorphic: true
+}); 
+// Alternatively, use an object for setup
+Picture.belongsTo('imageable', {
+  polymorphic: {
+    foreignKey: 'imageableId',
+    discriminator: 'imageableType'
+  }
+});
+```
+
+## HasOne polymorphic relations
+
+The relation definitions in 'HasOne' is almost same as 'HasMany', the only difference is `type` should be 'hasOne', for details, refer to HasMany polymorphic relations section.
```

Comment:
> The relation definitions in 'HasOne'

@kjdelisle I will rephrase it since it causes your confusion, while actually I explicitly describe it as **The relation definitions** is similar.
I am trying to emphasize user can use same definition parameters in the relation object(just change `type: hasMany` to `type: hasOne`), so I don't need to copy paste the whole definitions again.

Or maybe I can add hasMany & hasOne relation hyperlink here(those would be on a different page) to explain the essential difference between them:`one-to-many` vs `one-to-one`.

Any thought?